composer.json

```diff
@@ -180,6 +180,7 @@
         "drupal/config_inspector": "^2.0",
         "drupal/console": "^1.9",
         "drupal/core-dev": "^9",
+        "drupal/dev_mode": "^2.3",
```

Comment:
intentional? I don't see any config associated with it but just wanted to check since the project page has the standard "not in prod" warnings.

Reply:
Yes, is shotcut to debug twig